src/Log.cpp

```diff
@@ -51,6 +51,28 @@ void Log_Print(const char *_logBuffer, const uint8_t _minLogLevel, bool printTim
 	}
 }
 
+int Log_Printf(const uint8_t _minLogLevel, const char *format, ...) {
+	char loc_buf[201];	// Allow a maximum buffer of 200 characters in a single log message
+
+	int len;
+	va_list arg;
+	va_start(arg, format);
+    
+    // use the local buffer and trunctate string if it's larger
+    len = vsnprintf(loc_buf, sizeof(loc_buf), format, arg);
+
+	Log_Print(loc_buf, _minLogLevel, true);
+	if(len > sizeof(loc_buf) - 1) {
+		// long string was trunctated
+		Log_Print("...", _minLogLevel, false);
+	}
+	Log_Print("\n", _minLogLevel, false);
+
+    va_end(arg);
```

Comment:
Indentation should use tabs instead of spaces.

Reply:
fixed